### PR TITLE
docs: fix broken links

### PR DESCRIPTION
**Pull request by KhudaDad414**
**Description**

- this PR is supposed to fix invalid links -> https://github.com/asyncapi/modelina/runs/6337599876?check_suite_focus=true

### CODE REVIEW

File: docs/interpretation_of_JSON_Schema.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ The following JSON Schema keywords are [merged](#Merging-models) with the alread
 ## Merging models
 Because of the recursive nature of the interpreter (and the nested nature of JSON Schema) it happens that two models needs to be merged together. 
 
-If only one side has a property defined, it is used as is, if both have it defined they are merged based on the following logic (look [here](./input_processing.md#Internal-model-representation) for more information about the CommonModel and its properties):
+If only one side has a property defined, it is used as is, if both have it defined they are merged based on the following logic (look  [here](./input_processing.md#Internal-model-representation) for more information about the CommonModel and its properties):
```
Comment (KhudaDad414) [inline docs/interpretation_of_JSON_Schema.md:54]: this file is missing and I am not sure where it is supposed to point. 🤔

Comment (magicmatatjahu) [reply docs/interpretation_of_JSON_Schema.md:54]: That file probably was removed. We have to wait for @jonaslagoni when he comes back from vacation to answer us.

**Comment by magicmatatjahu** [comment]
@KhudaDad414 Could you look on Jonas's suggestions? Thanks! :)

**Comment by magicmatatjahu** [comment]
@KhudaDad414 ping, ping 😄

**Comment by jonaslagoni** [comment]
This is changing old doc files and not relevant changes for `next`, closing the PR. If you find anything in the `next` branch don't hesitate to open another! 👍

**Pull request closed without merging**